osu.Game/Tests/Visual/OsuTestScene.cs

```diff
@@ -75,9 +75,9 @@ protected IAPIProvider API
         /// <remarks>
         /// In interactive runs (ie. VisualTests) this will use the user's database if <see cref="UseFreshStoragePerRun"/> is not set to <c>true</c>.
         /// </remarks>
-        protected RealmContextFactory ContextFactory => contextFactory.Value;
+        protected RealmAccess Access => contextFactory.Value;
```

Comment:
Not sure about this one, `Access` seems too non-specific. Would prefer `RealmAccess` (or maybe even `Realm`) if feasible.

Reply:
weird, that was rider alone.

Reply:
Should be resolved. I've gone with just `Realm` as it doesn't clash due to case.